modules/apps/dynamic-data-mapping/dynamic-data-mapping-data-provider-impl/src/main/java/com/liferay/dynamic/data/mapping/data/provider/internal/rest/DDMRESTDataProvider.java

```diff
@@ -641,6 +633,7 @@ private void _setMultiVMPool(MultiVMPool multiVMPool) {
 	@Reference
 	private JSONWebServiceClientFactory _jsonWebServiceClientFactory;
 
+	@Reference
```

Comment:
@JanisZhang You can't reference a portal cache. You should reference `MultiVMPool`, and create the portal cache in activate, and remove it in deactivate.

Reply:
@dantewang  Noted, Thanks.